src/Symfony/Component/Translation/PluralizationRules.php

```diff
@@ -35,7 +35,7 @@ public static function get($number, $locale/*, bool $triggerDeprecation = true*/
         $number = abs($number);
 
         if (3 > \func_num_args() || func_get_arg(2)) {
-            @trigger_error(sprintf('The "%s" class is deprecated since Symfony 4.2.', __CLASS__), \E_USER_DEPRECATED);
+            trigger_deprecation('symfony/translation', '4.2', 'The "%s" class is deprecated.', __CLASS__);
```

Comment:
Hmm... this PR targets 5.x, Deprecations for 4.x should have been deleted right?

Reply:
Yeah, But I assume that this specific message survived the major release to give users more time to update their translations to icu format. 

But that is just a guess.

Reply:
This filr has been added as a result of a bad merge in abf8010eaedf3c78605c6225679735d69f387772
It should be removed from 5.2

Reply:
Removed from 5.2 in 4a8c0b14db